### DISCUSSION TITLE
[Question] Help, SDDM always says login failed

### DISCUSSION THREAD

**Discussion by Nathaniel-Eubanks** [prasanthrangan/hyprdots#1551] [Q&A]
Whenever I login on SDDM (and yes I type my info in correctly) it always says Login Failed!, but if I go over to a different tty I can login just fine. As far as I can tell the SDDM default.conf is setup correctly so I'm really not sure why this is the case.

**Comment by fmolero69** [comment]
I don't know if my answer is of help.
Uninstall SDDM and clear the package cache plus configuration. and then you reinstall SDDM. Depending on what distribution you are using.
Then you do the installation again from HyDE.
I hope you can solve it

**Reply by Nathaniel-Eubanks** [reply]
Yeah I gotta be real, that didn't work, I've read through all the documentation I can find for Hyprdots, HyDE, and SDDM, I've gone through I think every forum thread online over this, and everything I've tried hasn't worked. I just deleted SDDM and called it a day lol, logging in through tty works for me.

**Comment by kRHYME7** [comment]
You got a custom keyboard layout?.

https://askubuntu.com/questions/993901/cannot-login-to-sddm-but-to-tty
